superset/assets/javascripts/explore/main.css

```diff
@@ -34,3 +34,32 @@
 .background-transparent {
   background-color: transparent !important;
 }
+
+a {
```

Comment:
this is pretty general, seems like it should go in the theme css file. 
https://github.com/airbnb/superset/blob/master/superset/assets/stylesheets/less/cosmo/bootswatch.less